tensorboard/plugins/beholder/beholder_plugin.py

```diff
@@ -96,18 +98,14 @@ def _serve_is_active(self, request):
     }
     return http_util.Respond(request, response, 'application/json')
 
-
   def _fetch_current_frame(self):
     path = '{}/{}'.format(self.PLUGIN_LOGDIR, shared_config.SUMMARY_FILENAME)
-
-    try:
-      frame = file_system_tools.read_tensor_summary(path).astype(np.uint8)
-      self.most_recent_frame = frame
-      return frame
-
-    except (message.DecodeError, IOError, tf.errors.NotFoundError):
-      return self.most_recent_frame
```

Comment:
Why are we removing most_recent_frame?  That's a behavior change (missing data now immediately goes blank instead of preserving the last frame and section info on screen) and it's not clear to me that this is better for users?  It doesn't seem necessary to fix the initialization logic; you can still keep most_recent_frame and if it's unset, then default to the no data PNG.

Reply:
Fixed.